apps/docs/src/app/core/component-docs/date-picker/date-picker-docs.component.html

```diff
@@ -74,7 +74,9 @@
 <fd-docs-section-title [id]="'combobox-reactiveForm'" [componentName]="'datePicker'">
     Date Picker in Reactive Form
 </fd-docs-section-title>
-<description>The date-picker component may also be used within Angular Reactive Forms.</description>
+<description>The date-picker component may also be used within Angular Reactive Forms. There is used
+    <code>[disableFunction]</code>, which controls the <code>valid</code> flag
```

Comment:
Maybe 
"<code>[disableFunction]</code> is used to control the <code>valid</code> flag"

Reply:
the `[disableFunction]` is controls the valid flag